From: SeongJae Park <sj@kernel.org>
Subject: selftests/damon: add execute permissions to test scripts
Date: Mon, 26 Aug 2024 20:03:30 -0700

Some test scripts are missing executable permissions.  It causes warnings
that make the test output unnecessarily verbose.  Add executable
permissions.

Link: https://lkml.kernel.org/r/20240827030336.7930-4-sj@kernel.org
Signed-off-by: SeongJae Park <sj@kernel.org>
Cc: Brendan Higgins <brendanhiggins@google.com>
Cc: David Gow <davidgow@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 0 files changed

diff --git a/tools/testing/selftests/damon/damon_nr_regions.py a/tools/testing/selftests/damon/damon_nr_regions.py
old mode 100664
new mode 100755
diff --git a/tools/testing/selftests/damon/damos_apply_interval.py a/tools/testing/selftests/damon/damos_apply_interval.py
old mode 100664
new mode 100755
diff --git a/tools/testing/selftests/damon/damos_quota_goal.py a/tools/testing/selftests/damon/damos_quota_goal.py
old mode 100664
new mode 100755
diff --git a/tools/testing/selftests/damon/damos_quota.py a/tools/testing/selftests/damon/damos_quota.py
old mode 100664
new mode 100755
diff --git a/tools/testing/selftests/damon/damos_tried_regions.py a/tools/testing/selftests/damon/damos_tried_regions.py
old mode 100664
new mode 100755
diff --git a/tools/testing/selftests/damon/debugfs_target_ids_pid_leak.sh a/tools/testing/selftests/damon/debugfs_target_ids_pid_leak.sh
old mode 100664
new mode 100755
diff --git a/tools/testing/selftests/damon/debugfs_target_ids_read_before_terminate_race.sh a/tools/testing/selftests/damon/debugfs_target_ids_read_before_terminate_race.sh
old mode 100664
new mode 100755
diff --git a/tools/testing/selftests/damon/sysfs_update_schemes_tried_regions_hang.py a/tools/testing/selftests/damon/sysfs_update_schemes_tried_regions_hang.py
old mode 100664
new mode 100755
diff --git a/tools/testing/selftests/damon/sysfs_update_schemes_tried_regions_wss_estimation.py a/tools/testing/selftests/damon/sysfs_update_schemes_tried_regions_wss_estimation.py
old mode 100664
new mode 100755
_
